src/app/cli/src/coda_worker_testnet.ml

```diff
@@ -200,19 +198,25 @@ let start_prefix_check logger workers events testnet ~acceptable_delay =
     (Linear_pipe.iter events ~f:(function `Transition (i, (prev, curr)) ->
          Linear_pipe.write all_transitions_w (prev, curr, i) ))
 
-type user_cmd_status = {snapshots: int option array; passed_root: bool array}
+type user_cmd_status =
+  {snapshots: int option array; passed_root: bool array; result: unit Ivar.t}
 
 let start_payment_check logger root_pipe payment_pipe workers testnet
     ~acceptable_delay =
   let root_lengths = Array.init (List.length workers) ~f:(Fn.const 0) in
   let user_commands_under_inspection = Hashtbl.create (module User_command) in
-  let add_to_user_commands_under_inspection user_cmd =
+  let add_to_user_commands_under_inspection (user_cmd, result) =
     let snapshots =
       Array.init (List.length workers) ~f:(fun i ->
           if Api.synced testnet i then Some root_lengths.(i) else None )
     in
+    Logger.debug logger ~module_:__MODULE__ ~location:__LOC__
+      !"user command %{sexp:User_command.t} is added and snapshots are \
```

Comment:
use interpolation and metadata instead of formatting in sexps.

example: https://github.com/CodaProtocol/coda/blob/e795982a718cda00a282873da10ec52626f56595/src/lib/transition_handler/validator.ml#L88

Reply:
changed as requested